app/controllers/languages_controller.rb

```diff
@@ -13,6 +14,7 @@ def show
     @html_id = 'page'
     @body_id = 'languages'
     @locale  = Locale.find_by(slug: canonical_locale.canonical)
+    @title = PageTitle.new @locale.name
```

Comment:
```suggestion
    @title = PageTitle.new "#{@locale.name} / #{@locale.name_in_english} (#{@locale.slug})"
```

Could you add the name in English and the slug to the title too, please?